trace.go

```diff
@@ -1,5 +1,7 @@
 package runn
 
+const traceHeaderName = "X-Runn-Trace"
```

Comment:
This constant is the default value, correct?

```suggestion
const defaultTraceHeaderName = "X-Runn-Trace"
```